redis/client.py

```diff
@@ -2733,15 +2733,26 @@ def xpending_range(self, name, groupname, min, max, count,
         max: maximum stream ID.
         count: number of messages to return
         consumername: name of a consumer to filter by (optional).
+        idle: available from  version 6.2. filter entries by their
+        idle-time, given in milliseconds (optional).
         """
         pieces = [name, groupname]
         if min is not None or max is not None or count is not None:
             if min is None or max is None or count is None:
                 raise DataError("XPENDING must be provided with min, max "
                                 "and count parameters, or none of them. ")
+            if idle is not None:
```

Comment:
```suggestion
            try:
                if count < -1:
                    raise DataError("XPENDING count must be an integer >=1 -1")
                pieces.extend(['IDLE', str(idle)])
            except TypeError:
                pass
```

Reply:
Does not handle the case then `idle` is not an integer.